department-of-veterans-affairs/caseflow
# Fixed the focus indicator on the docket type badge

Resolves [CASEFLOW-1671](https://vajira.max.gov/browse/CASEFLOW-1671)

### Description
Please explain the changes you made here.

### Acceptance Criteria

### Testing Plan

1. Shadow user `BVASYELLOW`: http://localhost:3000/test/users
2. Navigate to the hearing schedule: http://localhost:3000/hearings/schedule
3. Click the “Schedule Veteran” button at the top left
4. Select the St. Petersburg, FL RO from the dropdown
5. Click the “AMA Veterans Waiting” tab and click the name of one of the Veterans
6. On the Case Details page, click the “Schedule Veteran” option from the task list
7. Tab to the docket number and ensure that the focus indicator wraps the docket type badge

### User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="1377" alt="Screen Shot 2021-06-28 at 12 52 29 PM" src="https://user-images.githubusercontent.com/61989526/123695888-f61ae980-d80f-11eb-88c9-dcd5fbe2569c.png">|<img width="1334" alt="Screen Shot 2021-06-28 at 12 49 17 PM" src="https://user-images.githubusercontent.com/61989526/123695922-0206ab80-d810-11eb-8a64-89d22ee7fb5b.png">
